packages/view/backend/sirius-components-view-diagram-edit/src/main/resources/icons/full/obj16/DropNodeTool.svg

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
Same icon as for drop tool, does it need to be duplicate ?

Reply:
Changed to use https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Apan_tool%3A